### PR TITLE
Facgame

### PR DESCRIPTION
**Pull request by y-zaky**
Created Case Study for FacGame

### CODE REVIEW

File: internal-projects/FacGame.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
+## Title of project
```
Comment (jsms90) [inline internal-projects/FacGame.md]: Delete this and make "Gunboat Diplomacy" an actual heading?

File: internal-projects/FacGame.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+GunBoat Diplomacy
+
+## Screencast
```
Comment (jsms90) [inline internal-projects/FacGame.md]: You can remove this heading. It will be obvious that it is a screencast

File: internal-projects/FacGame.md [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+![](https://user-images.githubusercontent.com/25886395/30488410-2083ca58-9a2e-11e7-9cf6-62416a2f3273.gif | width=150)
```
Comment (jsms90) [inline internal-projects/FacGame.md]: This link is broken

File: internal-projects/FacGame.md [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+
+## Challenge
+
+The main challenge was the fact none of our team members new anything about any of the technologies that we were to use. In fact initially we had no idea how we were going to build the game. However, with optimism and patience we were able to learn and deploy the technologies well.
+
+## Solution
+
+The solution was to use sockets.io to emit player position, velocity and rotation. We used the Three.JS library to build the scene, geometry and meshes. We used the CANNON.js library to build physics within the game.
+
+## Technologies
```
Comment (jsms90) [inline internal-projects/FacGame.md:22]: This needs formatting. Use the "View" button

File: internal-projects/FacGame.md [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+node.js
+heroku
+
+
+
+
+## Team members (with links to github profiles)
+[Rebeca](https://github.com/rebecacalvoquintero) , [Yahia](https://github.com/y-zaky), [James](https://github.com/RogeredBacon) and [Constantina](https://github.com/polyccon)
+
+## Quote from team member about client - NEW
```
Comment (jsms90) [inline internal-projects/FacGame.md]: hahahahha

maybe delete :wink:

File: internal-projects/FacGame.md [hunk 6]
```diff
@@ -0,0 +1,44 @@
...
+babelify(converts ES6 to ES5 for use of frontend files in bundles made with browserify)
+CANNON
+express
+node.js
+heroku
+
+
+
+
+## Team members (with links to github profiles)
```
Comment (jsms90) [inline internal-projects/FacGame.md]: Don't need the brackets

File: internal-projects/FacGame.md [hunk 7]
```diff
@@ -0,0 +1,44 @@
...
+[Rebeca](https://github.com/rebecacalvoquintero) , [Yahia](https://github.com/y-zaky), [James](https://github.com/RogeredBacon) and [Constantina](https://github.com/polyccon)
+
+## Quote from team member about client - NEW
+I love the client I don't want him to be frustrated anymore
+
+## link to URL (if public)
```
Comment (jsms90) [inline internal-projects/FacGame.md]: remove the brackets?

**Review by jsms90 — COMMENTED** [review]
[This is a good example](https://github.com/foundersandcoders/case-studies/blob/cf4b835acc14ac1ef17031f009363290caf51cb3/internal-projects/litlearn.md) for formatting

File: internal-projects/FacGame.md (hunk 2 above)
Comment (y-zaky) [reply internal-projects/FacGame.md]: True, thank you

**Pull request merged**